taipy/core/common/_check_dependencies.py

```diff
@@ -34,5 +31,10 @@ def _check_dependency_is_installed(module_name: str, package_name: str) -> None:
         )
 
 
-def _using_enterprise() -> bool:
-    return util.find_spec(_TAIPY_ENTERPRISE_MODULE) is not None
+class EnterpriseChecker:
```

Comment:
The official name is Enterprise Edition. Plus Checker is used in another context, to verify the consistency of config sections. I would propose the following names:
- EnterpriseEditionHelper
- EnterpriseEditionUtils
- EnterpriseEditionContext

What do you think?

Reply:
I will go with `EnterpriseEditionUtils` 😃